user/installing-dependencies.md

```diff
@@ -139,6 +139,34 @@ addons:
    - g++-4.8
 ```
 
+#### Identifying the source for missing package
+
+You see a message like this when you add `packages` but it is not found:
```

Comment:
I am not sure what you mean by this. With an example build log, or what? There are many lines between `Installing APT Packages (BETA)` and `E: Unable to locate package libcxsparse3.1.2`, and it is not practical to include them all.